jaxrs-api/pom.xml

```diff
@@ -414,6 +414,7 @@
                     <version>1.5</version>
                     <configuration>
                         <specMode>jakarta</specMode>
+                        <ignoreErrors>true</ignoreErrors>
```

Comment:
JAX-RS build will fail with spec-version-maven-plugin 1.5, for having [Bundle-SymbolicName=jakarta.ws.rs-api](https://github.com/eclipse-ee4j/jaxrs-api/issues/709) 
there is a [PR](https://github.com/eclipse-ee4j/glassfish-spec-version-maven-plugin/pull/6) to have it fixed.

Reply:
spec-version-maven-plugin version 2.0 is on OSSRH staging. It would be nice to update spec version plugin version number to 2.0 here too to test it.
We'll push plugin to central after making sure it works fine in few more projects.

Reply:
Also you can remove <ignoreErrors>true</ignoreErrors> after version update, it works fine with both javax and jakarta package prefixes.